codespell_lib/data/dictionary.txt

```diff
@@ -6438,13 +6455,24 @@ backslases->backslashes
 backslashs->backslashes
 backupped->backed-up, backed up,
 backwad->backwards
+backwar->backward
 backwardss->backwards
 backware->backward
+backwares->backwards
```

Comment:
Would `backwater`/`backwaters` be too far-fetched?

Reply:
`backwater(s)` is too uncommon.